src/styles/getMuiTheme.js

```diff
@@ -31,6 +31,8 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       textColor: palette.alternateTextColor,
       height: spacing.desktopKeylineIncrement,
       titleFontWeight: Typography.fontWeightNormal,
+      paddingLeft: spacing.desktopGutter,
+      paddingRight: spacing.desktopGutter,
```

Comment:
I guess not necessarily. single padding will work too. I will change it to just padding